docs/content/stable/api/ysql/exprs/func_setval.md

```diff
@@ -0,0 +1,91 @@
+---
+title: setval() function [YSQL]
+headerTitle: setval()
+linkTitle: setval()
+description: Set and return the value for the specified sequence.
+menu:
+  stable:
+    identifier: api-ysql-exprs-setval
+    parent: api-ysql-exprs
+isTocNested: true
+showAsideToc: true
+---
+
+
+## Synopsis
+
+Use the `setval(sequence_name , value, is_called)` function to set and return the value for the specified sequence.
```

Comment:
the `setval(sequence_name , value, is_called)` function  >

the `setval(sequence_name, value, is_called)` function